lib/tasks/mastodon.rake

```diff
@@ -33,6 +33,36 @@ namespace :mastodon do
     end
   end
 
+  desc 'Add a user by providing their email, username and initial password.' \
+       'The user will receive a confirmation email, then must reset their password before logging in.'
+  task add_user: :environment do
+    print 'Enter email: '
+    email = STDIN.gets.chomp
+
+    print 'Enter username: '
+    username = STDIN.gets.chomp
+
+    print 'Create user and send them confirmation mail (y/n): '
+    confirm = STDIN.gets.chomp
+    puts
+
+    if confirm == 'y'
+      user = User.new(email: email, password: SecureRandom.hex, account_attributes: { username: username })
+      if user.save
+        puts 'User added and confirmation mail sent to user\'s email.'
+        puts 'Note: After confirming their email, the user will need to reset their password to be able to login.'
```

Comment:
Just save the `SecureRandom.hex` into a variable and output it here. The admin will probably send the password to the user in question. This is pretty common practice for first logins (you get a random password, have to change it immediately). Better than having to reset password to get into the account.